### PR TITLE
V114

### PR DESCRIPTION
**Pull request by emthompson-usgs**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by emthompson-usgs** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@ requirements:
     - pyyaml
     - requests
     - scipy
+    - versioneer
```
Comment (ocefpaf) [inline recipe/meta.yaml:47]: You should not need `versioneer` as a dependency. Let me take a look upstream, maybe it is not configured correctly to write out the versions file with tags.

PS: versioneer is deprecated and project should consider moving to setuptools_scm.

Comment (emthompson-usgs) [reply recipe/meta.yaml:47]: Got it, thanks. I was just looking into trying to fix the bug I'm getting with versioneer. Instead I'll switch to setuptools_scm. We can ignore/close this PR given this distraction which isn't the primary issue right now...

Comment (ocefpaf) [reply recipe/meta.yaml:47]: BTW, this is the guide I follow when creating packages. It has an example on how to implement setuptools_scm as an alternative to versioneer.

Comment (emthompson-usgs) [reply recipe/meta.yaml:47]: Did you mean to link to a guide?

Comment (ocefpaf) [reply recipe/meta.yaml:47]: Yep, forgot the link. Sorry!

https://github.com/ioos/ioos-python-package-skeleton

**Comment by emthompson-usgs** [comment]
Gonna make a new release with the switch to setuptools_scm.

**Pull request closed without merging**